cmd/run.go

```diff
@@ -93,8 +93,8 @@ func (r *KoolRun) Execute(originalArgs []string) (err error) {
 // NewRunCommand initializes new kool stop command
 func NewRunCommand(run *KoolRun) (runCmd *cobra.Command) {
 	runCmd = &cobra.Command{
-		Use:   "run [SCRIPT]",
-		Short: "Runs a custom command defined at kool.yaml in the working directory or in the kool folder of the user's home directory",
+		Use:   "run [script]",
```

Comment:
maybe add `[args...]` here and mention the difference between single / multi line commands?

Reply:
at `exec` command we call this just `[command]`: https://github.com/kool-dev/kool/pull/285/files#diff-ce92eefe0908ac93cdc54bd33b7d01c347c159ae09a9324bd0479acdbc38c442R88

maybe better think and follow the standard.

Reply:
@dbpolito - I haven't yet addressed your comment about `[args...]` and single/multi line commands. But I did replace `run [script]` with `run [command]`, for consistency. Good catch!